apps/src/p5lab/spritelab/commands/behaviorCommands.js

```diff
@@ -10,6 +12,82 @@ export const commands = {
     return {func: func, name: func.funcName};
   },
 
+  burstFunc() {
+    return spriteArg => {
+      const sprite = this.getSpriteArray(spriteArg)[0];
```

Comment:
Recapping our conversation on Slack: We only assign these behaviors to a single sprite at a time, so we can leave this for now. This also maintains the convention used in other behaviors in this file. As a follow-up PR, we can add an additional guardrail. Because `getSpriteArray` could return an empty array, all of our behaviors could potentially throw an error, which might affect code execution.